Parser/ParserManager.php

```diff
@@ -9,28 +9,19 @@ class ParserManager
     /**
      * @var MarkdownParserInterface[]
      */
-    private $parsers = array();
+    private array $parsers = array();
```

Comment:
You may use short array syntax here too ;)

Reply:
I will take a look on this after the maintainers will give some feedback about this PR ;)

Reply:
Yeah I was just spotting it..

I hope some PR that brings in SF6 support will be merged some day soon :(